### PR TITLE
Add python-hawkular-client.rpm to oso-centos7-host-monitoring Dockerfile

### PR DESCRIPTION
**Pull request by zgalor**
Added the copr repo with the python-hawkular-client rpm and installed the rpm.
Tested locally

### CODE REVIEW

**Comment by zgalor** [comment]
@twiest  PTAL

**Comment by zgalor** [comment]
On second thought, it might be better to have the rpm in https://copr.fedorainfracloud.org/coprs/g/OpenShiftOnlineOps/openshift-tools/packages/

You don't wan't me to change the package externally and break your build.

@twiest @simon3z what do you think?

**Comment by twiest** [comment]
@zgalor The file you modified is a generated file (from a template). So the next time it's generated, your changes will be removed.

Here's more information on why we generate them:

https://github.com/openshift/openshift-tools/blob/prod/docker/README.adoc#directory-structure

We want to change this process because it's tedious and non-intuitive, but, for now it is our process.

You need to modify this Dockerfile and make sure it's in the `{% if base_os == "centos7" %}` section so that it only applies to centos:

https://github.com/openshift/openshift-tools/blob/prod/docker/oso-host-monitoring/src/Dockerfile.j2





As far as the "break the build", that's a good point, however, that also relegates us to maintaining that RPM and making sure it's up to date with the one we use in RHEL.

Neither is a good situation, but I think having you maintain a COPR repo with that client (at least for now) is probably the better of the two.

We can always change this if it becomes a problem.

**Comment by zgalor** [comment]
Whoops, my bad.
Fixed and generated from template.
@twiest PTAL

**Review by twiest — APPROVED** [review]
@zgalor Looks great!

Is this ready to merge?

**Comment by zgalor** [comment]
@twiest yes please

File: docker/oso-host-monitoring/src/Dockerfile.j2
Comment (twiest) [inline docker/oso-host-monitoring/src/Dockerfile.j2]: In docker, you ALWAYS need to clean up after yum so that it's cached files aren't committed to the image layer.

So this line should be something like this (untested):
```
RUN yum install -y python-hawkular-client && yum clean all
```


Also, do we want to do this install only on centos?

I believe the package is available for rhel as well. 

The copr repo addition, however, should only be in the centos section.

**Review by twiest — CHANGES_REQUESTED** [review]
Sorry, didn't think about this until after I approved the PR.

**Comment by zgalor** [comment]
Done. 
@twiest PTAL

**Comment by zgalor** [comment]
@twiest bump

**Comment by zgalor** [comment]
Actually, let's wait till I can consume the Hawkular rpm from Hawkular repo in copr

**Comment by twiest** [comment]
@zgalor ok, sounds good. Bump me again when that's ready.

**Comment by zgalor** [comment]
@twiest let's merge this, and once we have a hawkular repo on copr I'll submit a new PR.

**Comment by zgalor** [comment]
@twiest Now it's with the official Hawkular group repo.
Tested on my vm.

**Review by twiest — APPROVED** [review]
Looks great!

**Pull request merged**
